index.html

```diff
@@ -353,6 +353,35 @@ <h2>
               </tr>
             </tbody>
           </table>
+          <div class="note">
+            <p>Below are few non-normative recommendations when capturing display for 
+              applications that support full screen rendering.
+            </p>
+            <p>
+              For applications that support full screen rendering, like powerpoint for example, 
+              the decision of the display surface that gets captured while in full screen mode is 
+              specific to the user agent implementation. For consistency reasons, 
```

Comment:
Missing sentence.  "A user agent MAY choose to switch from capture of a window to the full screen when the associated application shows a full screen."